app/assets/javascripts/app/project/edit.js

```diff
@@ -29,3 +35,40 @@ App.addChild('ProjectEdit', _.extend({
   }
 }, Skull.Tabs));
 
+// FIXME: MORE DRY HERE !!
```

Comment:
Let's use another wrapper element that you can add to the edit template and use the same for all these actions ;)